config/sync/core.extension.yml

```diff
@@ -38,7 +39,6 @@ module:
   dropzonejs_eb_widget: 0
   dynamic_page_cache: 0
   editor: 0
-  editor_email_link: 0
```

Comment:
I think this needs to be added back in, correct @TaylorAkin ?

Reply:
Yea it does